test/cases/libvirt.sh

```diff
@@ -2,19 +2,21 @@
 set -euo pipefail
 
 # Get OS data.
-source /etc/os-release
-DISTRO_CODE="${DISTRO_CODE:-${ID}_${VERSION_ID//./}}"
+source /usr/libexec/osbuild-composer-test/set-env-variables.sh
 
 # Provision the software under test.
 /usr/libexec/osbuild-composer-test/provision.sh
 
 # Test the images
 /usr/libexec/osbuild-composer-test/libvirt_test.sh qcow2
 
-/usr/libexec/osbuild-composer-test/libvirt_test.sh openstack
+#TODO: remove this condition once there is rhel9 support for openstack and vhd image types
+if [[ "$DISTRO_CODE" != rhel-90 ]]; then
+  /usr/libexec/osbuild-composer-test/libvirt_test.sh openstack
+fi
 
 # RHEL 8.4 and Centos Stream 8 images also supports uefi, check that
-if [[ "$DISTRO_CODE" == "rhel_84" || "$DISTRO_CODE" == "rhel_85" || "$DISTRO_CODE" == "centos_8"  || "$DISTRO_CODE" == "rhel_90" ]]; then
+if [[ "$DISTRO_CODE" == "rhel-84" || "$DISTRO_CODE" == "centos-8" ]]; then
```

Comment:
Hmm, feels like we should bring back the rest of the versions with a hyphen, no ?

Reply:
yep